pages/science/reactors/idealgasconstpresmolereactor.rst

```diff
@@ -0,0 +1,62 @@
+.. title: Ideal Gas Constant Pressure Mole Reactor
+.. has_math: true
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Ideal Gas Constant Pressure
+      Mole Reactor</h1>
+
+   .. class:: lead
+
+      This page shows the derivation of the governing equations used in
+      Cantera's Ideal Gas Constant Pressure Mole Reactor model.
+
+      More information on the Ideal Gas Constant Pressure Mole Reactor class can
+      be found `here. <{{% ct_docs doxygen/html/de/daa/classCantera_1_1IdealGasConstPressureMoleReactor.html %}}>`__
+
+Ideal Gas Constant Pressure Reactor
+***********************************
+
+An Ideal Gas Constant Pressure Mole Reactor is defined by the two state variables:
+
+- :math:`T`, the temperature (in K)
+
+- :math:`n_k`, the number of moles for each species (in kmol)
+
+The energy equation in terms of temperature is necessary as we replaced enthalpy in the state vector with temperature.
+We develop the equation for temperature by writing the total enthalpy in terms of the molar enthalpy and moles of each species.
+
+.. math::
+
+   H = \sum_k \bar{h}_k(T) n_k(T)
+
+   \frac{dH}{dt} = \frac{dT}{dt}\sum_k n_k \bar{c_{p,k}} + \sum \bar{h}_k \dot{n}_k
+
+After some manipulations yields an equation for the
+temperature:
+
+.. math::
+
+   \frac{dT}{dt} = \frac{\dot{Q} - \sum \bar{h}_k \dot{n}_k}{\sum_k n_k \bar{c}_{p,k} }
+  \tag{1}
+
```

Comment:
For consistency with the RHS/LHS separation that we use, I would suggest keeping the $\sum_k n_k \bar{c}_{p,k}$ on the left hand side.